<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entrada</class>
 <widget class="QWidget" name="Entrada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>324</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>899</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Entrada</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../parkmanager.qrc">
    <normaloff>:/icons/parking.png</normaloff>:/icons/parking.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>40</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="corLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>655</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>655</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="horasLabel">
       <property name="text">
        <string>Hor√°rio:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="modeloLabel">
       <property name="text">
        <string>Modelo:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="corLabel">
       <property name="text">
        <string>Cor:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="modificarCkB">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLineEdit" name="modeloLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>655</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>655</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="diaLabel">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="horasLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="inputMask">
        <string>00:00:00; </string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="cursorPosition">
        <number>8</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="marcaLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>655</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>655</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="placaLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maxLength">
        <number>7</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="diaLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="inputMask">
        <string>00/00/0000; </string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="marcaLabel">
       <property name="text">
        <string>Marca:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="placaLabel">
       <property name="text">
        <string>Placa:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="novoButton">
       <property name="text">
        <string>&amp;Limpar</string>
       </property>
       <property name="icon">
        <iconset resource="../parkmanager.qrc">
         <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
       </property>
       <property name="shortcut">
        <string>F6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimirButton">
       <property name="text">
        <string>&amp;Imprimir</string>
       </property>
       <property name="icon">
        <iconset resource="../parkmanager.qrc">
         <normaloff>:/icons/document-print.png</normaloff>:/icons/document-print.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+P, F7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fecharButton">
       <property name="text">
        <string>&amp;Fechar</string>
       </property>
       <property name="icon">
        <iconset resource="../parkmanager.qrc">
         <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>F8</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>placaLE</tabstop>
  <tabstop>marcaLE</tabstop>
  <tabstop>modeloLE</tabstop>
  <tabstop>corLE</tabstop>
  <tabstop>novoButton</tabstop>
  <tabstop>imprimirButton</tabstop>
  <tabstop>fecharButton</tabstop>
 </tabstops>
 <resources>
  <include location="../parkmanager.qrc"/>
 </resources>
 <connections/>
</ui>
